config/locales/defaults/ca.yml

```diff
@@ -15,7 +15,7 @@ ca:
     - dv
     - ds
     abbr_month_names:
-    -
+    - 
```

Comment:
Maybe could we remove these empty values?

Reply:
This looks strange. I didn't change these files https://github.com/PopulateTools/gobierto/blob/fb23c24f82dcf53dd41ba6c65c97849f62e20f13/config/locales/defaults/ca.yml cc @ferblape @entantoencuanto

Reply:
It looks like you added a space, but you can ignore it, we can't remove the blank here, it's a Rails default.